### PR TITLE
Added the ability to import files before we construct the body

### PR DESCRIPTION
**Pull request by maraisr**
Adds the ability to have a

```
#import './data.json';
```

in the mock file - makes it easier when you have other unit tests reliying on some responses.

### CODE REVIEW

**Comment by maraisr** [comment]
@odino let me know when this PR can be merged, and if not, do you mind giving me owner rank? Wouldn't mind maintaining it.

**Comment by odino** [comment]
Hi @maraisr this is good to go -- can you update the readme though?

File: test/mockserver.js [hunk 1]
```diff
@@ -301,5 +301,12 @@ describe('mockserver', function() {
 
             assert.equal(res.status, 404);
         });
+
+        it('should be able to handle impoprts', function() {
```
Comment (razanbilwani) [inline test/mockserver.js]: `impoprts` > `imports`

**Comment by maraisr** [comment]
@odino done! Docs updated! <3

File: test/mockserver.js (hunk 1 above)
Comment (maraisr) [inline test/mockserver.js]: Good pick up - fixed.

**Comment by odino** [comment]
published on npm with `v1.6.0` -- thanks!

**Pull request merged**